modules/ossm-rn-new-features.adoc

```diff
@@ -30,6 +30,32 @@ This release of {SMProductName} addresses Common Vulnerabilities and Exposures (
 |Envoy Proxy
 |1.20.6
 
+|Jaeger
+|1.36
+
+|Kiali
+|1.48.2-1
+|===
+
+=== Copy route labels and annotations
+
+With this enhancement, you can copy specific labels or annotations for an OpenShift route. {SMProductName} copies all labels and annotations present in the Istio Gateway resource (with the exception of annotations starting with kubectl.kubernetes.io) into the managed OpenShift Route resource.
+
+== New features {SMProductName} version 2.2.1
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), bug fixes, and is supported on OpenShift Container Platform 4.9 and 4.10.
```

Comment:
Why doesn't this say "is supported on OpenShift Container Platform 4.9 or later." so that we don't have to update it when a new OCP release comes out?  (this would need to be updated in three places)

Reply:
Good point @JStickler. In one of the prior z releases we said supported on 4.9 and 4.10. I was carrying that model forward. I can make this change now.